src/main/play-doc/operation/ConsolidatedLogging.md

```diff
@@ -40,6 +40,28 @@ The provided Elasticsearch bundle configuration is using these settings:
 
 To change the settings create a new bundle configuration by modiying the `bundle.conf` file inside of the bundle configuration zip file. The configuration file is named `elasticsearch-prod-{digest}.zip` and is located in the `extra` folder as well. Afterwards reload the bundle with the new configuration.
 
+### Elasticsearch bundle rolling upgrade
+
+Elasticsearch stores its data within the filesystem. As such, to preserve existing data when performing deployment, it's important to ensure the data is replicated to the new instance of Elasticsearch.
+
+This can be achieved by following these steps.
+
+* Commission a new ConductR node where the new Elasticsearch instance is going to execute.
+* Start this new node and ensure this node is able to successfully join the existing ConductR cluster.
+* Turn off one of the old instance where Elasticsearch is running.
```

Comment:
Do I need to know which nodes ES is running on? Can I just shut down a node, wait long enough for it to go yellow if needed and move on? As an operator, I shouldn't have to be concerned with what bundle is running on each node. I might have to wait for data sync, but not worry about what's running where.